### ISSUE TITLE
Eclipse Che content on Docs Landing Page

### ISSUE THREAD

**Issue by micgibso** [eclipse/codewind#1758]
**Description of the enhancement:**
Enhance the new content of the Eclipse Che section of the landing page on the master branch. The new section comprises twelve topics:

- Installing Codewind for Eclipse Che
- Adding registries in Che
- Creating the Codewind workspace
- Setup to use Tekton Pipelines
- Set Docker registries
- Using Codewind in Theia
- OpenShift Do (odo) support in Codewind
- Docker Registry Guidance
- Using the OpenShift Registry
- Support for multiple users
- Installing Kibana and filtering logs in IBM Cloud Private
- Uninstalling Codewind for Eclipse Che

**Proposed solution:**
Edit these topics so the information flows, is grammatically correct, and conforms to standards.

**Comment by sishida** [comment]
Getting Started:
- Installing Codewind for Eclipse Che (grammar review)
- Configuring Codewind for Eclipse Che* (add new topic)
   - Adding Registries (combine existing topics of Adding Registries, Docker Registry Guidance, and Setting Docker Registries)
   - Creating the Workspace (grammar review, add paragraph about optional configurations, link)
- Creating your first project (add new topic. Refer to "Using Codewind in Theia" as its closest relative.)
- Making a code change (add new topic)

**Comment by MelHopper** [comment]
Hi @sishida re. the 'Configuring Codewind for Eclipse Che' topic, this will sit at the same level as Install, and will have the 'adding registries' and 'creating the workspace' as children:

Installing Codewind for Eclipse Che
Configuring Codewind for Eclipse Che
-> Adding Registries
-> Creating the workspace
Creating your first project
Making a code change

**Comment by MelHopper** [comment]
Hi @sishida we're missing the 'Configuring Codewind for Eclipse Che' title, under which the children 'Adding a container registry' and Creating a Codewind workspace in Che'.

**Comment by MelHopper** [comment]
and a Next Step section that sends user to 'Creating a Codewind workspace in Che' - are you able to add?

**Comment by sishida** [comment]
@MelHopper Please see this issue for the limitations regarding child topics: https://github.com/eclipse/codewind/issues/2380

**Comment by MelHopper** [comment]
Can you now delete che-usingtheia.md _'Using codewing from the Che Theia IDE'_? - has the content been moved into the new topic _Creating your first project_ ?

**Comment by MelHopper** [comment]
Hi Sarah, are you happy to create a new issue to cover the remaining work 'Need to add paragraph about optional configurations, link)(eclipse/codewind-docs#315)'? - then we can close this work item and it is not blocked by github issue https://github.com/eclipse/codewind/issues/2381 'Move Optional topics out of Eclipse Che tutorial block and into it's own block'

**Comment by sishida** [comment]
Closed this issue because we can cover remaining work in https://github.com/eclipse/codewind/issues/2381.
